### ISSUE TITLE
Fingerprint not working — Samsung A7 2018 (A750FN)

### ISSUE THREAD

**Issue by Yuuji9029** [phhusson/treble_experimentations#580]
Hi, the Fingerprint won't work anymore on the latest GSIs, I tried on AOSP 9.0 v114, and also on the Descendant GSI and HavocOS GSI. The enrollment either never completes or gives "unknown error" on lockscreen.

`ro.vendor.build.fingerprint:`samsung/a7y18lteser/a7y18lte:9/PPR1.180610.011/A750FNPUU1BSD2:user/release-keys 
[Right now I'm on Descendant ThreeDotZero GSI]

The attached log is from vince31fr's Havoc 2.6 GSI.
[logcat_06-09-2019_18-39-32.txt](https://github.com/phhusson/treble_experimentations/files/3317781/logcat_06-09-2019_18-39-32.txt)

**Comment by phhusson** [comment]
what do you mean anymore? What's the last known working?

**Comment by Yuuji9029** [comment]
@phhusson It was working on an older Version of AOSP 9.0, it was v107 or v109 I think.

**Comment by phhusson** [comment]
did you change vendor /Stock ROM?

**Comment by Yuuji9029** [comment]
@phhusson yeah I updated to the latest Pie Stock ROM, which has the April security patch

**Comment by phhusson** [comment]
Right that's more likely what caused the issue

Do you have dm-verity disabled? If so, edit /vendor/build.prop change
ro.vendor.build.fingerprint to have Samsung at the start instead of samsung
and see if it works better

**Comment by Yuuji9029** [comment]
@phhusson I'm rooted with latest Magisk, so yes dm-verity should be disabled afaik.

Another Problem is that I can't edit the vendor build.prop, it says "failed to save build.prop" when editing, although editing other files (like the system build.prop) works just fine :/

**Comment by Jayanth-L** [comment]
Hello @Yuuji9029 
Is the tethering and hotspot working on your device 
I have a750fn.

**Comment by Yuuji9029** [comment]
@Jayanth-L Yes, Tethering / Hotspot is working, but only with 2,4G band afaik.
